packages/Card/Card.jsx

```diff
@@ -71,9 +74,37 @@ const Card = ({ variant, children, fullHeight, spacing, ...rest }) => {
     spacingProps.inset = 4
   }
 
+  const ContainerStyles = imageProps =>
+    imageProps &&
+    imageProps.position &&
+    handleResponsiveStyles({ position: imageProps.position }, ({ position }) => {
+      if (!imageProps) return {}
+      const direction = {
+        left: 'row',
+        right: 'row-reverse',
+        top: 'column',
+        bottom: 'column-reverse',
+        none: 'row',
+      }
+      const styles = {
+        display: 'flex',
+        flexDirection: direction[position],
+        justifyContent: 'space-between',
+        '> img': {
```

Comment:
I feel iffy of this `'> img'` selector.

If we can guarantee  that the only child of the `styled-component` will be the `fullBleedImage` `Image`  I would feel much more confident.

Reply:
We can guarantee that it will only select the full bleed image. This is a direct descendent selector, and there are only two direct descendents: the full bleed image, and a box with all the children. There will never be another image direct descendent in this div.